src/frontend/src/components/Post/telescope-post-content.css

```diff
@@ -2,6 +2,9 @@
   font-size: 1.8rem;
   line-height: 1.6;
   text-align: left;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  hyphens: auto;
```

Comment:
Will doing this on everything cause odd behaviour in code/pre?  We probably want to allow long lines there, but not in other post content.  Should we "undo" this on pre?

Reply:
@humphd I try to test on <code> and <pre> and I did not spot any odd behavior. But may be there are some special cases that can cause odd behavior? 
I definitely can unset for <code> and <pre> but I can't be 100% sure that it will work because as I said I don't see any odd behavior.